packages/paste-core/components/tabs/src/Tabs.tsx

```diff
@@ -1,42 +1,55 @@
 import * as React from 'react';
 import * as PropTypes from 'prop-types';
+import type {BoxProps} from '@twilio-paste/box';
 import {Flex} from '@twilio-paste/flex';
-import {useTabPrimitiveState, TabPrimitiveInitialState, TabPrimitiveStateReturn} from '@twilio-paste/tabs-primitive';
+import {Box} from '@twilio-paste/box';
+import {useTabPrimitiveState} from '@twilio-paste/tabs-primitive';
+import type {TabPrimitiveInitialState, TabPrimitiveStateReturn} from '@twilio-paste/tabs-primitive';
 import {TabsContext} from './TabsContext';
-import {Variants} from './types';
+import type {Variants} from './types';
 
 export interface TabStateReturn extends TabPrimitiveStateReturn {
   [key: string]: any;
 }
 
 export interface TabsProps extends TabPrimitiveInitialState {
   children?: React.ReactNode;
+  element?: BoxProps['element'];
   state?: TabStateReturn;
   variant?: Variants;
 }
 
 // Set orientation to horizontal because undefined enables all arrow key movement
 const Tabs = React.forwardRef<HTMLDivElement, TabsProps>(
-  ({children, orientation = 'horizontal', state, variant, ...initialState}, ref) => {
-    const tab = state || useTabPrimitiveState({orientation, ...initialState});
-    const value = React.useMemo(() => ({...tab, variant}), [...Object.values(tab), variant]);
+  ({children, element = 'TABS', orientation = 'horizontal', state, variant, ...initialState}, ref) => {
+    // If returned state from primitive has orientation set to undefined, use the default "horizontal"
+    const {orientation: tabOrientation = orientation, ...tab} =
+      state || useTabPrimitiveState({orientation, ...initialState});
+
+    const value = React.useMemo(() => ({...tab, orientation: tabOrientation, variant}), [
+      ...Object.values(tab),
+      orientation,
```

Comment:
should this be `tabOrientation`?

Reply:
Yes, thank you! I'm kinda surprised I didn't get a lint error here 🤔